src/pymor/tools/frozendict.py

```diff
@@ -39,6 +41,8 @@ def __reduce__(self):
 class SortedFrozenDict(FrozenDict):
     """A sorted immutable dictionary."""
 
+    __slots__ = ()
```

Comment:
Isn't this redundant?

Reply:
No, it's actually required to repeat that in every subclass, which is why I also added it to `Parameters` now.

Reply:
So `__slots__` are actually inherited, but subclasses still get  a `__dict__`. That I did not know (and find somehwat surprising)